test/functional/rpc_psbt.py

```diff
@@ -11,6 +11,8 @@
 import json
 import os
 
+import IPython
```

Comment:
que?

Reply:
whoop lol

was using this for debugging. Removed.